python_modules/jupedsim/jupedsim/models/collision_free_speed.py

```diff
@@ -53,24 +56,40 @@ class CollisionFreeSpeedModelAgentParameters:
     Attributes:
         position: Position of the agent.
         time_gap: Time constant that describe how fast pedestrian close gaps.
-        v0: Maximum speed of the agent.
+        desired_speed: Maximum speed of the agent.
         radius: Radius of the agent.
         journey_id: Id of the journey the agent follows.
         stage_id: Id of the stage the agent targets.
     """
 
     position: tuple[float, float] = (0.0, 0.0)
     time_gap: float = 1.0
-    v0: float = 1.2
+    desired_speed: float = 1.2
     radius: float = 0.2
     journey_id: int = 0
     stage_id: int = 0
 
+    def __init__(
+        self,
+        v0=None,
+        **kwargs,
```

Comment:
Nope we cannot do it like this, now unknown parameters are silently ignored, e.g.
`CollisionFreeSpeedModelAgentParameters(foo=True)` was an error and is now silently ignored.